📜Remove hardcoded dotnet version reference (#321)
KinsonDigital/VelaptorDocs: docs/guides/guides/project-setup.mdx

## <span className="color-step">Step 1: Create the application</span>

> Create a new C# console application using dotnet version specified in the `EmptyProject.csproj`, look after property group `<TargetFramework>`.

Reviewer: Please make the following changes:

```suggestion
Create a new C# console application using dotnet version specified in the `EmptyProject.csproj`, look for the `<TargetFramework></TargetFramework>` XML tag.
```

Author: That makes sense, good point out XML tag also newbie friendly 🌟 I should have put that there in the beginning.

Reviewer: Np at all.
